packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -3623,6 +3623,93 @@ void main() {
     );
   });
 
+  testWidgets(
+    'Can drag the left handle while the right handle disappears',
```

Comment:
Nit: Maybe this is more clear, but it's up to you: "while the right handle disappears" => "while the right handle remains off-screen"

Reply:
Same thing for the other test with a similar description below.